examples/wsgi.py

```diff
@@ -1,6 +1,12 @@
 """
-An exemplifying Apache python WSGI to a Devicehub app with a dispatcher.
+[Testing] Apache python WSGI to a Devicehub app with a dispatcher.
 """
+from ereuse_devicehub.config import DevicehubConfig
 from ereuse_devicehub.dispatchers import PathDispatcher
 
-application = PathDispatcher()
+class MyConfig(DevicehubConfig):
+    # SQLALCHEMY_DATABASE_URI = 'postgresql://usody:UhcmFB33dvuhqG@localhost/devicehub_testing'
+    SQLALCHEMY_DATABASE_URI = 'postgresql://{}:{}@{}/{}'
```

Comment:
What about using a [named arguments](https://docs.python.org/3.6/library/string.html#format-examples) to help readability and make easier to render the template?
```python
>>> 'Coordinates: {latitude}, {longitude}'.format(latitude='37.24N', longitude='-115.81W')
'Coordinates: 37.24N, -115.81W'
>>> coord = {'latitude': '37.24N', 'longitude': '-115.81W'}
>>> 'Coordinates: {latitude}, {longitude}'.format(**coord)
'Coordinates: 37.24N, -115.81W'
```